### PR TITLE
lua script: clean up / channel fix / optimization

### PR DESCRIPTION
**Pull request by PichotM**

### CODE REVIEW

**Comment by Itokoyamato** [comment]
Hey, thanks for the PR ! I'll check it out and try to merge it to master tomorrow.
Just curious, do you have any data on how much perf is affected by this ? If it changes at all

**Comment by PichotM** [comment]
None, but this one is the best in terms of optimizations https://github.com/Itokoyamato/TokoVOIP_TS3/pull/20/commits/542033b32a542fe2d6d392cd65a03b8d5f2cf370
instead of calling 256 times NetworkIsPlayerActive you immediatly get the list of "active players"

**Comment by PichotM** [comment]
Two new commits:
https://github.com/Itokoyamato/TokoVOIP_TS3/pull/20/commits/b3732815d3a6344abb70390c94b870a9e7b58fdb GET_ACTIVE_PLAYERS returns server id so it needed to be changed

https://github.com/Itokoyamato/TokoVOIP_TS3/pull/20/commits/d27e68417c487ca69d15d6a315ff68f24ff8f626 tiny optimization and avoid useless calls

**Comment by Itokoyamato** [comment]
It appears your latest changes broke things. Seems like some reference to the index of the loop you replaced.
There's also a conflict with master with my latest update :p

**Comment by PichotM** [comment]
conflict is fixed, what the last thing broken?

File: fivem_script/tokovoip_script/src/c_main.lua [hunk 1]
```diff
@@ -18,9 +18,133 @@ local targetPed;
...
+			if (not mode or (mode ~= 1 and mode ~= 2 and mode ~= 3)) then mode = 1 end;
+			local volume = -30 + (30 - dist / voip.distance[mode] * 30);
+			if (volume >= 0) then
+				volume = 0;
+			end
+			--
+			local angleToTarget = localHeading - math.atan(playerPos.y - localPos.y, playerPos.x - localPos.x);
+
+			-- Set player's default data
+			usersdata[i] = {
```
Comment (Itokoyamato) [inline fivem_script/tokovoip_script/src/c_main.lua]: Several references to i, but your replaced the loop by playerServerId

Comment (PichotM) [reply fivem_script/tokovoip_script/src/c_main.lua]: oh sorry, I thought it was already done

**Comment by PichotM** [comment]
it should be better now
using this kind of insertion `usersdata[#usersdata + 1] = tbl` is also better in terms of performance
also in lua table like this `local tbl = { 1, 2, 3, 4, 5, 6 }` are better than `local tbl = { [2] = 2, [5] = 1 }`

**Pull request merged**